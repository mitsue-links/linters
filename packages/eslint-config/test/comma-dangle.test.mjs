import path from 'path';
import {lintFile} from './helper.mjs';

test('comma-dangle_ok.mjs', async () => {
    const filePath = 'comma-dangle_ok.mjs';
    const data = await lintFile(path.join('fixtures', filePath));

    expect(data).toStrictEqual([{
        filePath,
        messages: [],
    }]);
});

test('comma-dangle_error.mjs', async () => {
    const filePath = 'comma-dangle_error.mjs';
    const data = await lintFile(path.join('fixtures', filePath));

    expect(data).toMatchSnapshot();
});
