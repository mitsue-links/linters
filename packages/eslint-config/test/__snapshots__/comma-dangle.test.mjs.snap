// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comma-dangle_error.mjs 1`] = `
[
  {
    "filePath": "comma-dangle_error.mjs",
    "messages": [
      {
        "line": 12,
        "ruleId": "comma-dangle",
        "severity": 2,
      },
      {
        "line": 18,
        "ruleId": "comma-dangle",
        "severity": 2,
      },
      {
        "line": 30,
        "ruleId": "comma-dangle",
        "severity": 2,
      },
    ],
  },
]
`;
